### ISSUE TITLE
Discord Server Verification

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#4275]
## Description
We need to update our Discord API key. This may not be without complication, as we don't have good documentation over how that was originally set up, and the main account used for logging here appears to be locked out. 

## Context
[Bot Verification and Data Allowlisting](https://support.discord.com/hc/en-us/articles/360040720412-Bot-Verification-and-Data-Whitelisting#how-do-i-start-the-process)

**Comment by alexyoung23j** [comment]
Appears that the problem is just that our bot needs to be verified. Starting that process

**Comment by timolegros** [comment]
@CowMuon can we close this issue if we are verified? I have not encountered this issue during any of my recent Discobot work.

**Comment by CowMuon** [comment]
Good catch Tim, this is an old ticket. Confirming the live Discobot is not affected.
